flink-optimizer/pom.xml

```diff
@@ -58,6 +58,12 @@ under the License.
 			<artifactId>guava</artifactId>
 			<version>${guava.version}</version>
 		</dependency>
+		<dependency>
```

Comment:
This dependency should be removed, it should be unnecessary. If it is actually necessary, then there is an issue somewhere else ;-)